### ISSUE TITLE
Ledger error for new records (30.12.1)

### ISSUE THREAD

**Issue by dragonjet** [KC3Kai/KC3Kai#1655]
still existing on 30.12.1
ledger values are 0 for new records.
as reported on facebook

**Comment by LostLogia4** [comment]
Do we have any idea on the possible cause, or at least, how to investigate them?

**Comment by dragonjet** [comment]
I'll ask further from the reporter

**Comment by LostLogia4** [comment]
Console didn't report anything unusual. Seems like we'll need to ask people like Javran to debug this.

**Comment by LostLogia4** [comment]
I've used Javran's debugging code to look up the entries and printed out a huge amount of entries:
```
KC3Database.get_lodger_data(Range(0,Infinity,0,1), function(t){t.forEach(function(b){ if(b.type.startsWith("")){console.log(b);} })});
```

And here's the list of datatype:
```
exped
sortie
quest
dsitem
critem
pvp
rmditem
crship
akashi
dsship
remodel
useitem
lbas
```
PVP appears to be an extra datatype that was unlisted in the ledger, critem and dsitem has no number suffixes

**Comment by sinsinpub** [comment]
nothing useful detail offered, cannot even say there is bug. maybe just because he did not record any lbas data, maybe just no data in his selected time range 😂

**Comment by LostLogia4** [comment]
Wait, you need to get new LBAS entries to fix this bug?

**Comment by sinsinpub** [comment]
where is the bug? i mean there may be no any problem at all 😓 
my ledger are not 0 for new records (he mean lbas?), so what he means: 
![image](https://cloud.githubusercontent.com/assets/160240/21149394/4fcec086-c196-11e6-85f7-c3bd99e9f398.png)

**Comment by sinsinpub** [comment]
write some memo on this discussion: Types of consuming not recorded by ledger for now:
 * LBAS sortie fuel and ammo consume (no api response to client found at all, amount may be calculated)
 * LBAS losses of fuel and bauxite on enemy air raid (also no api, even amount of losses not known)
 * Jet bomber steel consume (also no api, amount is being tested)

**Comment by LostLogia4** [comment]
It's still all 0 on my side, and I had kept my IndexedDB since last year, so it might've been problem with my old entries. (how did you edit the Indexed DB anyways?)

**Comment by sinsinpub** [comment]
😮  you mean all your data shown as 0 no matter what time range selected? hmm, it's hard to say a reason. a possible one is mass records causing array overflow. or your pc time causing wrong data recorded or  unexpected query condition...
if there is no exception on js console, shortcut way may be getting your IndexedDB to debug...

**Comment by dragonjet** [comment]
he sent me this XD

http://puu.sh/sNUxQ/948b2dbb7c.png

**Comment by LostLogia4** [comment]
@dragonjet The same thing happened to me, and the f12 console doesn't return anything unusual.

**Comment by sinsinpub** [comment]
u mean console show nothing even these timing profiling?
![image](https://cloud.githubusercontent.com/assets/160240/21166151/a9c02e16-c1de-11e6-8741-9703179b4a3a.png)

**Comment by LostLogia4** [comment]
Console only displays the following:

objects.js:4 KC3改 Messengers loaded
Master.js:1 Loaded master: raw data
global.js:3 Synchronous XMLHttpRequest on the main thread is deprecated because of its detrimental effects to the end user's experience. For more help, check https://xhr.spec.whatwg.org/.send @ global.js:3
RemodelDb.js:1 RemodelDb: no update required
allstrategytabs.js:26 Async Completed 13261 26505291047 29461

**Comment by dragonjet** [comment]
same thing, seems there's nothing on his console

![chrome_2016-12-14_13-27-44](https://cloud.githubusercontent.com/assets/557312/21170609/3ef61e7e-c201-11e6-91c1-e01106f7ca65.png)

**Comment by LostLogia4** [comment]
Seems like we couldn't debug this much further unless if we get to the bottom of the indexedDB. Problem is I've already accumulated a vast amount of records over the year, so it way larger than notepad++ could handle (it's ~50MB of text for crying out loud)

**Comment by mikenzb** [comment]
Not sure if this has been fixed in #1734 but will close for now, reopen if issue still persists.

**Comment by LostLogia4** [comment]
It was never fixed for me, so I had to discard my sortie records from Fall 2015 to March 2017.
